modules/dnn/src/layers/pooling_layer.cpp

```diff
@@ -359,7 +364,7 @@ class PoolingLayerImpl CV_FINAL : public PoolingLayer
             size_t stripeStart = r.start*stripeSize;
             size_t stripeEnd = std::min(r.end*stripeSize, total);
             int kernel_w = kernel.width, kernel_h = kernel.height;
-            int pad_w = pad.width, pad_h = pad.height;
+            int pad_left = pad_l, pad_top = pad_t, pad_right = pad_r, pad_bottom = pad_b;
```

Comment:
I think it's not necessary. Just use origin variables below.

Reply:
done